app/main.py

```diff
@@ -1,13 +1,33 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        error_detail_name: (
+            error["line_number"] if error_detail_name == "line" else
+            error["column_number"] if error_detail_name == "column" else
+            error["text"] if error_detail_name == "message" else
+            error["code"] if error_detail_name == "name" else "flake8"
+        )
+        for error_detail_name in [
+            "line", "column", "message", "name", "source"
+        ]
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
```

Comment:
simplier way:

`    return {"errors": [format_linter_error(error) for error in errors],
            "path": file_path,
            "status": "failed" if errors != [] else "passed"
            }`

Reply:
Noted. Thank for the tip.